pkg/netpol/connlist/conns_formatter_dot.go

```diff
@@ -78,9 +62,11 @@ func (d formatDOT) writeOutput(conns []Peer2PeerConnection) (string, error) {
 	edgeLines := make([]string, len(conns))  // list of edges lines
 	peersVisited := make(map[string]bool, 0) // acts as a set
 	for index := range conns {
-		edgeLines[index] = getEdgeLine(conns[index])
-		externalPeersLines = categorizeAndAddPeerLine(conns[index].Src(), peersVisited, externalPeersLines, nsPeers)
-		externalPeersLines = categorizeAndAddPeerLine(conns[index].Dst(), peersVisited, externalPeersLines, nsPeers)
+		c := conns[index]
+		connStr := common.ConnStrFromConnProperties(c.AllProtocolsAndPorts(), c.ProtocolsAndPorts())
+		edgeLines[index] = dotformatting.GetEdgeLine(c.Src().String(), c.Dst().String(), connStr, "gold2", "darkgreen")
```

Comment:
I suggest to add consts for color names as: `connlistEdgeColor = "gold2"` and` edgeFontColor = "darkgreen"`

Reply:
Done